src/app/modules/submission/encrypt-submission/encrypt-submission.service.ts

```diff
@@ -323,6 +324,27 @@ export const getEncryptedSubmissionData = (
   })
 }
 
+export const getAllEncryptedSubmissionData = (formId: string) => {
+  return ResultAsync.fromPromise(
+    EncryptSubmissionModel.find({
+      form: formId,
+      submissionType: SubmissionType.Encrypt,
+    }),
+    (error) => {
```

Comment:
to sort. and limit to first 1000
also need frontend UI to inform admin that only first 1000 resps will be used for chart

Reply:
a04084db7 for BE. Still missing prompt for FE.